index.bs

```diff
@@ -1018,78 +1045,83 @@ The <dfn>lifetime</dfn> of a
 [=/transaction=] is as follows:
 
 1. A transaction is <dfn>created</dfn> with a [=transaction/scope=] and a [=transaction/mode=].
-    When a transaction is created its [=transaction/active flag=] is initially set.
-
-1. The implementation must allow [=requests=] to be [=request/placed=]
-    against the transaction whenever the [=transaction/active flag=] is set. This
-    is the case even if the transaction has not yet been [=transaction/started=].
-    Until the transaction is [=transaction/started=] the implementation must not
-    execute these requests; however, the implementation must keep
-    track of the [=requests=] and their order. Requests may be placed
-    against a transaction only while that transaction is [=transaction/active=].
-    If an attempt is made to place a request against a transaction
-    when that transaction is not [=transaction/active=], the implementation must
-    reject the attempt by throwing a "{{TransactionInactiveError}}" {{DOMException}}.
-
-1. Once an implementation is able to enforce the constraints defined
-    for the transaction [=transaction/scope=] and [=transaction/mode=], defined below, the
-    implementation must [=queue a task=] to <dfn
-    lt="start|started">start</dfn> the transaction asynchronously.
-
-1. Once the transaction has been [=transaction/started=] the implementation can
-    start executing the [=requests=] placed against the transaction.
-    Unless otherwise defined, requests must be executed in the order
-    in which they were made against the transaction. Likewise, their
-    results must be returned in the order the requests were placed
-    against a specific transaction. There is no guarantee about the
-    order that results from requests in different transactions are
-    returned. Similarly, the transaction [=transaction/modes=] ensure that two
-    requests placed against different transactions can execute in any
-    order without affecting what resulting data is stored in the
-    database.
+    When a transaction is created its [=transaction/state=] is initially [=transaction/active=].
+
+1. When an implementation is able to enforce the constraints defined for the transaction [=transaction/scope=] and [=transaction/mode=], defined [below](#transaction-scheduling], the implementation must [=queue a task=] to <dfn lt="start|started">start</dfn> the transaction asynchronously.
```

Comment:
Tab has been advocating for line breaking at logical (sentence, clause) boundaries rather than to just wrap at 80 columns and reflowing, since that makes future diffs easier to parse. That said, there are some places this could break.